Azure/azure-cli
# [ARM] az group export: Add new parameters `--skip-resource-name-params` and `--skip-all-params` to support skip parameterization

**Description**  

- Add --skip-resource-name-params argument which removes the use of parameters from name properties
- Add --skip-all-params argument which removes the use of all parameters from the response
- Add 

--resource-ids argument to filter the results based on a string containing whitespace-separated 

**Testing Guide**  
- To export all resource templates from a resource group without resource name parameterization: **az group export --name contoso_rg --skip-resource-name-params --resource-ids "*"**
- To export all resource templates from a resource group without parameters: **az group export --name contoso_rg --skip-all-params --resource-ids "*"**

---

This checklist is used to make sure that common guidelines for a pull request are followed.
